### PR TITLE
Convert a Package file for Swift 4

### PR DESCRIPTION
**Pull request by lightwave**
Provide a package file with Swift 4 syntax

### CODE REVIEW

File: Package@swift-4.swift [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// swift-tools-version:4.0
+import PackageDescription
+
+let package = Package(
+    name: "FluentProvider",
+    products: [
+        .library(name: "FluentProvider", targets: ["FluentProvider"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/vapor/fluent.git", .upToNextMajor(from: "2.0.0")),
```
Comment (0xTim) [inline Package@swift-4.swift]: Can you mark fluent as from 2.4.0

File: Package@swift-4.swift [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        .package(url: "https://github.com/vapor/vapor.git", .upToNextMajor(from: "2.0.0")),
```
Comment (0xTim) [inline Package@swift-4.swift]: And Vapor should be from 2.1.0

**Review by 0xTim — CHANGES_REQUESTED** [review]
Can you update the CI scripts as well to test and build on Swift 4 as well as Swift 3?

**Comment by 0xTim** [comment]
Also, whilst I’m being pedantic, could you update the README to point to Swift 4 as well?

**Comment by lightwave** [comment]
Updated and pushed changes.

Thanks
--
Chris

**Review by 0xTim — APPROVED** [review]

**Comment by 0xTim** [comment]
LGTM - @tanner0101 ?

**Comment by tanner0101** [comment]
thanks :)

**Comment by lightwave** [comment]
Tim, I missed the CI scripts, but where is the CI scripts in fluent-provider… I’m not seeing it.

Thanks
--
Chris

**Comment by 0xTim** [comment]
They can be found here https://github.com/vapor/fluent-provider/blob/master/circle.yml

They already run on Swift 4, but it’s probably worth doing another PR to also run the tests in 3.1, like the main repo - https://github.com/vapor/vapor/blob/master/circle.yml

**Pull request merged**